mozartdata/transforms/dim/old_to_new_sku_map.sql

```diff
@@ -0,0 +1,37 @@
+/*
+ Purpose: Map old skus to new skus and new upcs
+ Granularity: One row per old sku
+ Primary key column: old_sku
+ Joining:
+    To old products: use column old_sku to sku
+    To new products: use column new_sku to sku
+ Notes: This data is human input. Please consider this when troubleshooting.
+ */
+
+select
+    upc.display_name -- Not recommended for joining but feasible for some tables
+  , upc.old_sku -- PRIMARY KEY, most recommended for joining
+  , upc.category
+  , upc.tier
+  , upc.due_date_for_new_skus as new_sku_due_date
+  , upc.new_skus              as new_sku -- Can be used to join new skus in product tables
+  , upc.new_upcs              as new_upc
+  , upc.po_placement
+  , upc.estimated_arrival
```

Comment:
These should be cast as date, and called date
```suggestion
  , date(upc.po_placement) as po_placement_date
  , date(upc.estimated_arrival) as estimated_arrival_date
```